### ISSUE TITLE
How can I access the previous version of this package?

### ISSUE THREAD

**Issue by jyp256** [mlampros/fastText#4]
Hi. I used the previous version of this package which was named as "fastTextR"?
Now that name seems to be taken up by another author.
I need the previous version of the package to prepare my replication data.
Looking forward to updates on this issue. Thank you.

Addendum: Particularly, I want to run the following code:
list_params = list(input = PATH_INPUT, output = PATH_OUT, #input = file.path(tempdir(), "doc.txt"), output = tempdir(),
                     command = 'skipgram', lr = 0.02, lrUpdateRate = 100,
                     dim = 400, ws = 5, epoch = 10, minCount = 1, neg = 5, 
                     wordNgrams = 1, loss = "hs", bucket = 2e+06,
                     minn = 0, maxn = 0, thread = 8, t = 1e-04, verbose = 2)
  
  vecs = fasttext_interface(list_params, # file.path(tempdir(),"model_logs.txt")
                            MilliSecs = 100)

But I get the following error message using the current version:
Error: "PATH_OUT.bin" cannot be opened for saving.

**Comment by mlampros** [comment]
hi @jyp256, would you like to include a reproducible example? If the code that you include in your message is from a website then feel free to add the URL

**Comment by jyp256** [comment]
Hi. Thank you for following up. I solved the issue using “devtools::install_github” function.
Thanks!


Sent from Yahoo Mail for iPhone


On Tuesday, September 5, 2023, 12:25 AM, Lampros Mouselimis ***@***.***> wrote:




hi @jyp256, would you like to include a reproducible example? If the code that you include in your message is from a website then feel free to add the URL

**Comment by mlampros** [comment]
good to hear you found the solution to the issue
